src/main/java/BasketballGames/Coach.java

```diff
@@ -0,0 +1,10 @@
+package BasketballGames;
+
+import java.time.LocalDate;
+
+public class Coach extends Human {
```

Comment:
If a class extends, but does nothing, you could simply treat a coach as human. It has nothing extra.